DEPENDENCIES.md

```diff
@@ -527,7 +526,6 @@ graph LR;
   npm-->pacote;
   npm-->parse-conflict-json;
   npm-->proc-log;
-  npm-->proggy;
```

Comment:
`proggy` was previously required in `display.js` as a placeholder for how progress messages could be rendered, but not actually used. I opted to remove it as we now are showing a spinner without any specific messages. A future iteration that should useful progress messages would bring this dependency back.

Reply:
d9cf7fcf16950c1bb885cdc99d3ba9e9c400e7e1